StyleCop.Analyzers/Directory.Build.targets

```diff
@@ -2,16 +2,16 @@
 <Project>
 
   <ItemGroup>
-    <!-- Enable code generation for resource files -->
+    <!-- Enable code generation for resource files. -->
     <CodeGenerationEmbeddedResource Include="@(EmbeddedResource)" Exclude="**\*.??-??.resx" />
```

Comment:
tip: resource files can also just end with `*.??.resx`. In my revision based on yours, I ended up with this:

```xml
	<ItemGroup>
		<!-- Enable code generation for resource files. -->
		<CodeGenerationEmbeddedResource Include="@(EmbeddedResource)" Exclude="**\*.??-??.resx;**\*.??.resx" />
		<EmbeddedResource Update="@(CodeGenerationEmbeddedResource)" />
		<AdditionalFiles Include="@(CodeGenerationEmbeddedResource)" />

		<NonCodeGenerationEmbeddedResource Include="@(EmbeddedResource)" Exclude="@(CodeGenerationEmbeddedResource)" />
		<EmbeddedResource Update="@(NonCodeGenerationEmbeddedResource)" />
	</ItemGroup>

	<!-- Special handling for embedded resources to show as nested in Solution Explorer -->
	<ItemGroup>
		<!-- Localized embedded resources are just dependent on the parent RESX -->
		<EmbeddedResource Update="**\*.??-??.resx" DependentUpon="$([System.IO.Path]::ChangeExtension($([System.IO.Path]::GetFileNameWithoutExtension(%(Identity))), '.resx'))" />
		<EmbeddedResource Update="**\*.??.resx" DependentUpon="$([System.IO.Path]::ChangeExtension($([System.IO.Path]::GetFileNameWithoutExtension(%(Identity))), '.resx'))" />
	</ItemGroup>
```

Reply:
This entire block is going to go away once the ResxSourceGenerator update merges.